.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.10"] # min and max supported versions?
```

Comment:
Unit tests run on min and max python versions you want to support

Reply:
Look good, I hope it works on python 3.7. But I think I tried to only use features of 3.6 actually, but I lost track of it at this point.